### PR TITLE
Cert Auth

### PR DESCRIPTION
**Pull request by RichardSmedley**
DOC-8250

### CODE REVIEW

**Review by daschl — CHANGES_REQUESTED** [review]
Note that this is not certificate authentication! Cert auth is the other way round - where you provide a certificate to the server instead of user/password credentials to authenticate.

Here, we are not dealing with authentication, but the client is accepting the server certificate for the connection (you still have to provide username and password). So this is just "encrypted / tls  connections with the custom certificate the server provides)

**Comment by RichardSmedley** [comment]
>  ...So this is just "encrypted / tls connections with the custom certificate the server provides)

Oops, thanks for catching that @daschl - fixed now.

File: modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 package com.couchbase.devguide;
 
-import com.couchbase.client.core.deps.io.netty.handler.ssl.util.InsecureTrustManagerFactory;
+// import com.couchbase.client.core.deps.io.netty.handler.ssl.util.InsecureTrustManagerFactory; // use in testing if you use .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)) instead of Cert Auth
```
Comment (daschl) [inline modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java:19]: still nothing to do with cert auth (comment)

File: modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java [hunk 2]
```diff
@@ -43,7 +43,8 @@ public static void main(String... args) {
 
         ClusterEnvironment env = ClusterEnvironment.builder()
                 .securityConfig(SecurityConfig.enableTls(true)
-                        .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))
+                        // .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))  // use in testing if you use .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)) instead of Cert Auth (below)
```
Comment (daschl) [inline modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java:46]: same

Comment (daschl) [reply modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java:46]: more accurate would be "used in testing if you want to trust all server certificates  / do not validate them"

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 3]
```diff
@@ -121,6 +121,9 @@ Couchbase Cloud Sample::
 +
 --
 If you are connecting to https://docs.couchbase.com/cloud/index.html[Couchbase Cloud], be sure to get the correct endpoint as well as user, password, and `couchbasecloudbucket`  -- and see the <<cloud-connections, Cloud section>>, below.
+During development, you may wish to use `.securityConfig(SecurityConfig.enableTls(true)` (commented out in the code sample below),
```
Comment (daschl) [inline modules/hello-world/pages/start-using-sdk.adoc:124]: this does not make sense? enable tls is always needed both in dev and prod, I think you wanted to mention the insecure manager?

**Comment by RichardSmedley** [comment]
Closing as has been done properly elsewhere.

**Pull request closed without merging**